endpoints.json

```diff
@@ -25,5 +25,32 @@
         }
       ]
     }
+  },
+  "GET /api/articles/:article_id": {
+    "description": "serves an array of all articles of a given article_id",
+    "exampleResponse": {
+        "articles": [
```

Comment:
should we be sending back an array of articles or one article object?

Reply:
Thanks saima - please check the updated pull request :)